contracts/transmuter/Storage.sol

```diff
@@ -64,70 +66,70 @@ enum WhitelistType {
+    uint8 isManaged; // If the collateral is managed through external strategies
+    uint8 isMintLive; // If minting from this asset is unpaused
+    uint8 isBurnLive; // If burning to this asset is unpaused
+    uint8 decimals; // IERC20Metadata(collateral).decimals()
+    uint8 onlyWhitelisted; // If only whitelisted addresses can burn or redeem for this token
+    uint216 normalizedStables; // Normalized amount of stablecoins issued from this collateral
+    uint64[] xFeeMint; // Increasing exposures in [0,BASE_9[
+    int64[] yFeeMint; // Mint fees at the exposures specified in `xFeeMint`
+    uint64[] xFeeBurn; // Decreasing exposures in ]0,BASE_9]
+    int64[] yFeeBurn; // Burn fees at the exposures specified in `xFeeBurn`
+    bytes oracleConfig; // Data about the oracle used for the collateral
+    bytes whitelistData; // For whitelisted collateral, data used to verify whitelists
+    ManagerStorage managerData; // For managed collateral, data used to handle the strategies
 }
 
 struct TransmuterStorage {
-    IAgToken agToken;                            // agToken handled by the system
-    uint8 isRedemptionLive;                      // If redemption is unpaused
-    uint8 statusReentrant;                        // If call is reentrant or not
-    uint128 normalizedStables;                   // Normalized amount of stablecoins issued by the system
-    uint128 normalizer;                          // To reconcile `normalizedStables` values with the actual amount
-    address[] collateralList;                    // List of collateral assets supported by the system
-    uint64[] xRedemptionCurve;                   // Increasing collateral ratios > 0
-    int64[] yRedemptionCurve;                    // Value of the redemption fees at `xRedemptionCurve`
-    mapping(address => Collateral) collaterals;  // Maps a collateral asset to its parameters
-    mapping(address => uint256) isTrusted;       // If an address is trusted to update the normalizer value
+    IAgToken agToken; // agToken handled by the system
+    uint8 isRedemptionLive; // If redemption is unpaused
+    uint8 statusReentrant; // If call is reentrant or not
+    uint128 normalizedStables; // Normalized amount of stablecoins issued by the system
+    uint128 normalizer; // To reconcile `normalizedStables` values with the actual amount
+    address[] collateralList; // List of collateral assets supported by the system
+    uint64[] xRedemptionCurve; // Increasing collateral ratios > 0
+    int64[] yRedemptionCurve; // Value of the redemption fees at `xRedemptionCurve`
+    mapping(address => Collateral) collaterals; // Maps a collateral asset to its parameters
+    mapping(address => uint256) isTrusted; // If an address is trusted to update the normalizer value
     mapping(address => uint256) isSellerTrusted; // If an address is trusted to sell accruing reward tokens
```

Comment:
let's add a comment here for the fact that the role can help for other stuff as well in fact